docs/pages/api/routes.md

```diff
@@ -91,6 +91,18 @@ Update scene definition and audio mixer configuration for output with ID `:outpu
 
 ***
 
+### Request keyframe
+
+```http
+POST: /api/output/:output_id/request_keyframe
+```
+
+```typescript
+type RequestBody = {}
+```
+
+Requests additional keyframe (I frame) on the video output. It doesn't guarantee that the next output frame will be a keyframe.
```

Comment:
> next output frame will be a keyframe.

Is this correct? Wouldn't be more correct to say next packet? Personally, I would just drop that second sentence.